testing/travis.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import base64
+import json
+import logging
+import os
+import os.path
+import socket
+import ssl
+import subprocess
+import sys
+import urllib2
+
+# FROM plugins/inventory/terraform.py
+# PYTHON_PATH is added to in .travis.yml
+import terraform as tf
+
+## TRAVIS STEPS
+
+def install():
+    cmds = [
```

Comment:
No need to make another variable here, this can be simplified to 
```python
run_cmds([
    (["pip", "install", "-r", "requirements.txt"], 1),
    (["curl", "-SL0", "terraform.zip",  "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip"], 1),
    (["unzip", "-d", "testing/cache/", "terraform.zip"], 1),
    (["ssh-keygen", "-N", '""', "-f", "~/.ssh/id_rsa"], 1),
])
```